CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 - [#122](https://github.com/HewlettPackard/oneview-puppet/issues/122) Uri_parsing should support upper case for uri
 - [#133](https://github.com/HewlettPackard/oneview-puppet/issues/133) Allow set a timeout for Image_streamer_golden_image upload
 - [#132](https://github.com/HewlettPackard/oneview-puppet/issues/132) Allow option force for Image_streamer_golden_image download operations
+- [#139](https://github.com/HewlettPackard/oneview-puppet/issues/139) Error running get_default_settings (Logical Interconnect Group/Synergy)
```

Comment:
How about a more descriptive name for the issue? 
Something like, 'Error running `oneview_logical_interconnect`'s ensure method `get_default_settings` after upgrading the OneView SDK to version 4.1'